src/lavinmqctl.cr

```diff
@@ -255,14 +261,56 @@ class LavinMQCtl
   private def handle_response(resp, *ok)
     return if ok.includes? resp.status_code
     if resp.status_code == 503
-      puts "[ERROR] #{resp.body}"
+      output resp.body
       exit 2
     end
-    puts "#{resp.status_code} - #{resp.status}"
-    puts resp.body if resp.body? && !resp.headers["Content-Type"]?.try(&.starts_with?("text/html"))
+    output "#{resp.status_code} - #{resp.status}"
+    output resp.body if resp.body? && !resp.headers["Content-Type"]?.try(&.starts_with?("text/html"))
     exit 1
   end
 
+  private def output(data, columns = nil)
+    if @options["format"]? == "json"
+      data.to_json(STDOUT)
+      puts
+    else
+      case data
+      when Hash, NamedTuple
+        data.keys.each do |k|
+          puts "#{k}: #{data[k]}"
+        end
+      when Array
+        output_array(data, columns)
+      else
+        puts data
+      end
+    end
+  end
+
+  private def output_array(data : Array, columns : Array(String)?)
+    if columns
+      puts columns.join("\t")
+    else
+      if first = data.first?
+        first = first.as_h if first.is_a?(JSON::Any)
+        puts first.keys.join("\t")
```

Comment:
```suggestion
        puts first.each_key.join(STDOUT, "\t")
```

Reply:
`each_key` is expected to be invoked with a block, so this throws an error. 
Maybe just `puts first.keys.join(STDOUT, "\t")` instead?

Reply:
What types may `data` contain? `Hash#each_key` returns an iterator (if not invoked with a block), but `NamedTuple#each_key` only has one overload that requires a block.

Reply:
`data.first` will either be a `NamedTuple` or a `JSON::Any`

Reply:
so maybe something like this? 
```
        if first.is_a?(NamedTuple)
          puts first.keys.join(STDOUT, "\t")
        elsif first.is_a?(JSON::Any)
          puts first.as_h.each_key.join(STDOUT, "\t")
        end
      end
```